### PR TITLE
add fesev-ap4

### PR DESCRIPTION
**Pull request by FFHener**
Added another EAP225-outdoor

### CODE REVIEW

File: group_vars/location_fesev/networks.yml [hunk 1]
```diff
@@ -36,14 +37,17 @@ networks:
 location__channel_assignments_11a_standard__to_merge:
   fesev-ap1: 36-40
   fesev-ap2: 40-40
+  fesev-ap4: 60-40
 
 location__channel_assignments_11g_standard__to_merge:
   fesev-ap1: 1-20
   fesev-ap2: 6-20
   fesev-ap3: 13-20
+  fesev-ap4: 11-20
```
Comment (PolynomialDivision) [inline group_vars/location_fesev/networks.yml:46]: I would not use 11, if you already use 13?

Comment (FFHener) [reply group_vars/location_fesev/networks.yml]: I  didnt wanted to touch the other APs. In the way they're installed they shoudn't interfere. But i have to update them soonish anyway, so will shift the channels

**Comment by PolynomialDivision** [comment]
Please squash that together into one or multiple commits and delete the merge commit. ;)

**Comment by FFHener** [comment]
Done. I also realised i didnt added the fesev-ap4 in host_vars, its now added as well.

**Comment by PolynomialDivision** [comment]
lgtm :)

**Pull request merged**